src/core/Form/Textarea/Textarea.md

```diff
@@ -1,11 +1,17 @@
 ```js
 import { Textarea } from 'suomifi-ui-components';
+import React from 'react';
 
+const testiReffi = React.createRef();
```

Comment:
I'd also change this testi riffi to just be const ref, const test, etc. Maybe use ref for all of these?